packages/angular_devkit/build_angular/src/webpack/configs/styles.ts

```diff
@@ -158,6 +158,11 @@ export function getStylesConfig(wco: WebpackConfigOptions): webpack.Configuratio
   }
 
   const { supportedBrowsers } = new BuildBrowserFeatures(wco.projectRoot);
+  const postcssPresetEnvPlugin = postcssPresetEnv({
```

Comment:
You can just push to `extraPostcssPlugins`.

Reply:
I was contemplating that but went with this, as I felt `extraPostcssPlugins` was more for conditional plugins. Anyway, I just tried pushing to `extraPostcssPlugins` but then the types are incompatible; presumably because `postcssPresetEnv` actually depends on postcss 7.